include/ibamr/FESurfaceDistanceEvaluator.h

```diff
@@ -97,6 +97,11 @@ class FESurfaceDistanceEvaluator
      */
     void mapIntersections();
 
+    /*!
+     * Data structure to map the face normal of the elements, common vertices and edges.
```

Comment:
This comment should be 
`\brief Compute the face normal of the surface elements.`